portlets/src/main/webapp/vue-app/realizations/components/RealizationItem.vue

```diff
@@ -15,7 +15,7 @@ along with this program; if not, write to the Free Software Foundation,
 Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 -->
 <template>
-  <tr>
+  <tr :id="realization.id">
```

Comment:
to use a more specific id:
```suggestion
  <tr :id="`GamificationRealizationItem${realization.id}`">
```